aytests/cms.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e -x
+
+# checking if the keys have been downloaded
+grep "dummy" /etc/salt/pki/minion/minion.pem
+grep "dummy" /etc/salt/pki/minion/minion.pub
```

Comment:
the test contains ```default.key``` and ```default.pub```. I know nothing about salt, so should I expect that these two files are these ```minion.*``` files?